src/rgw/rgw_log.cc

```diff
@@ -357,6 +357,23 @@ int rgw_log_op(RGWRados *store, RGWREST* const rest, struct req_state *s,
   else
     set_param_str(s, "HTTP_REFERER", entry.referrer);
   set_param_str(s, "REQUEST_URI", entry.uri);
```

Comment:
Are we unnecessarily setting `entry.uri` here ? Correct me if I'm wrong.

Reply:
nice catch!